diff --git a/src/QProblem.c b/src/QProblem.c
index 4bb5b92..22601b5 100644
--- a/src/QProblem.c
+++ b/src/QProblem.c
@@ -469,7 +469,7 @@ void QProblemCPY(	QProblem* FROM,
 	OptionsCPY( &(FROM->options),&(TO->options) );
 	QProblem_setPrintLevel( TO,TO->options.printLevel );
 
-	ConstraintsCPY( &(FROM->constraints),&(TO->constraints) );
+	ConstraintsCPY( (FROM->constraints), (TO->constraints) );
 
 	DenseMatrixCPY(FROM->A, TO->A);
 
